test/unit/spec/v2/view-builder/utils/AuthenticatorUtil_spec.js

```diff
@@ -33,56 +13,36 @@ describe('v2/utils/AuthenticatorUtil', function() {
       ],
       'errorSummary': ''
     };
-    const result = removeRequirementsFromError(errorJSON, policy);
+    const result = removeRequirementsFromError(errorJSON);
     expect(result).toEqual({
       'errorCauses': [
         {
           'property': 'credentials.passcode',
-          'errorSummary': 'Password requirements were not met.'
+          'errorSummary': 'Password requirements were not met'
         }
       ],
       'errorSummary': ''
     });
   });
 
-  it('does not change anything if there are no requirements in error message', function() {
-    const policy = {
-      'complexity': {
-        'minLength': 8,
-        'minLowerCase': 1,
-        'minUpperCase': 1,
-        'minNumber': 1,
-        'minSymbol': 1,
-        'excludeUsername': true,
-        'excludeAttributes': [
-          'firstName',
-          'lastName'
-        ],
-        'excludeFirstName': true,
-        'excludeLastName': true
-      },
-      'age': {
-        'minAgeMinutes': 120,
-        'historyCount': 4
-      }
-    };
+  it('does not change anything if the error is not "requirements missing" error message', function() {
     const errorJSON = {
       'errorCauses': [
         {
           'property': 'credentials.passcode',
           'errorSummary': [
-            'Password requirements were not met.'
+            'This password was found in a list of commonly used passwords. Please try another password.'
```

Comment:
Is this the UX approved message? We should suppress the reasoning and just say it's a weak password right ?

Reply:
So password gets multiple error messages depending on the scenario. I am just improving the test to a real world scenario where the user may get `This password was found in a list of commonly used passwords. Please try another password.`. This is what is displayed in a certain scenario as of today.

This PR addresses the requirements-not-met scenario where the actual requirements need to be trimmed off and a simplified message will be displayed.

Reply:
@pradeepdewda-okta - I actually added mock for this error and updated i18n for it in this PR.